statistics-production/ud.qmd

```diff
@@ -817,6 +817,77 @@ for dashboards and other secondary statistics services, whilst also helping with
 
 ---
 
+## Sex and gender
+
+The Department's policy on collecting sex and gender data is that statistics 
+should preferentially be presented for sex rather than gender. These terms, as 
+used in statistics publications, are defined in the following sections and 
+publication teams should be careful not to mislabel sex categories as gender or
+vice-versa.
+
+---
+
+### Sex
+
+---
+
+Sex as reported by DfE statistics, is:
+
+> This is a value which identifies the sex of a person as recognised in law, 
```

Comment:
This doesn't follow on from the lead in - would it read better if you dropped the 'This is' from the start for the quote?

Reply:
Updated